### PR TITLE
Feature/logout

### PR DESCRIPTION
**Pull request by juliancoleman**
@jadengore neat little addition

### CODE REVIEW

**Comment by juliancoleman** [comment]
wait, whoa, what?

**Comment by juliancoleman** [comment]
I don't know what happened here, but it looks like this was branched from feature/login with all its commit history @jadengore

File: vigor/templates/vigor/signup.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {% load staticfiles %}
 <link rel="stylesheet" href="{% static 'vigor/style.css' %}">
+<<<<<<< HEAD
+{% include "vigor/header.html" %}
+=======
```
Comment (ebramanti) [inline vigor/templates/vigor/signup.html]: @juliancoleman Uh oh some merge issues 😢

File: vigor/views.py [hunk 2]
```diff
@@ -50,12 +51,20 @@ def signup(request):
         form = UserCreationFormExtended()
     return render(request, 'vigor/signup.html', {'form': form})
 
+@login_required
```
Comment (ebramanti) [inline vigor/views.py:54]: @juliancoleman YES!

File: vigor/templates/vigor/header.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<header>
+  <nav>
+    <ul>
+      <li><a href="#">Vigor</a></li>
+      <li><a href="https://juliancoleman.github.io/docs">Docs</a></li>
+      <li><a href="https://github.com/juliancoleman/vigor">Github</a></li>
+    </ul>
+    {% if not user.is_authenticated %}
```
Comment (ebramanti) [inline vigor/templates/vigor/header.html:8]: @juliancoleman Maybe an if-else for these two cases?

File: vigor/templates/vigor/signup.html (hunk 1 above)
Comment (juliancoleman) [inline vigor/templates/vigor/signup.html]: fixed

File: vigor/templates/vigor/header.html (hunk 3 above)
Comment (juliancoleman) [inline vigor/templates/vigor/header.html:8]: that makes much more sense

**Comment by ebramanti** [comment]
@juliancoleman 🍆 and 🚢, squashing your commits should get rid of the ugly merge commits you had to add 😄

**Pull request merged**